third-party/realdds/py/pyrealdds.cpp

```diff
@@ -593,16 +578,24 @@ PYBIND11_MODULE(NAME, m) {
         .def( "set_description", &dds_option::set_description )
         .def( "to_json", []( dds_option const & self ) { return self.to_json().dump(); } );
 
-    using realdds::dds_stream_format;
-    py::class_< dds_stream_format >( m, "stream_format" )
+    using realdds::dds_video_encoding;
+    py::class_< dds_video_encoding > video_encoding( m, "video_encoding" );
+    video_encoding  //
         .def( py::init<>() )
         .def( py::init< std::string const & >() )
-        .def( "is_valid", &dds_stream_format::is_valid )
-        .def( "to_rs2", &dds_stream_format::to_rs2 )
-        .def( "from_rs2", &dds_stream_format::from_rs2 )
-        .def( "__nonzero__", &dds_stream_format::is_valid )  // Called to implement truth value testing in Python 2
-        .def( "__bool__", &dds_stream_format::is_valid )     // Called to implement truth value testing in Python 3
-        .def( "__repr__", []( dds_stream_format const & self ) { return self.to_string(); } );
+        .def( "is_valid", &dds_video_encoding::is_valid )
+        .def( "to_rs2", &dds_video_encoding::to_rs2 )
+        .def( "from_rs2", &dds_video_encoding::from_rs2 )
+        .def( "__nonzero__", &dds_video_encoding::is_valid )  // Called to implement truth value testing in Python 2
```

Comment:
Do we still support python 2?

Reply:
No... I will add task to remove Python 2 support